rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -564,7 +565,131 @@ namespace rsx
 			return true;
 		}
 
-		//Get intersecting set - Returns all objects intersecting a given range and their owning blocks
+		std::vector<std::pair<u32, u32>> subtractive_intersect(std::vector<section_storage_type*> marked_sections, std::vector<section_storage_type*> sections_to_exclude)
+		{
+			std::vector<std::pair<u32, u32>> result;
+			bool intersection_exists = false;
+
+			result.reserve(marked_sections.size());
+
+			for (const auto &section : marked_sections)
+			{
+				result.push_back(section->get_protected_range());
+			}
+
+			for (const auto &dirty : sections_to_exclude)
+			{
+				const auto intersect_test = dirty->get_protected_range();
```

Comment:
I suggest changing this to `exclude_range`, `exclude_start` and `exclude_end` for readability.